### ISSUE TITLE
Explanation and formulas for combination of mixers

### ISSUE THREAD

**Issue by Morlacus** [DeviationTX/deviation-manual#26]
I Propose the following changes for simpler formulas and a better understanding

Instead of 
The combination of mixers for a given output channel is defined by the Mux type:

proposed
The combination Cx of mixers 1 to n  for a given output channel is defined by the Mux type:

Change proposed for formulas and headings

The formula could take into account one multiply mux only for a better understanding. All muxes do not need to be multiply.
So the formula could refer to the [value resulting from mixers n-1 to 1] and the change by a multiply Mn. 

So instead of

For a ‘Multiply’ mux:
Cx = if(\ |Sn|\ ) {\ |Mn|\ } else {1} * if (\ |Sn-1|\ ) {\ |Mn-1|\ } else {1} * … * if (\ |S0|\ ) {\ |M0|\ } else {1}

I propose

For a Mn ‘Multiply’ mux activated by switch Sn:
x = if(\ |Sn active|\ ) {\ |Mn|\ } else {1} * [value resulting from mixers n-1 to 1]

The same process could be used for every formula ( Replace, multply, add, Max and Min)

**Comment by TheRealMoeder** [comment]
Since this section is probably not understood by non programmers or hobby mathematics in either version, I personally don't really care since I can happily understand both :) I wonder if we should elaborate more on complex mixers, perhaps with one or two examples for non-nerds :P

**Comment by Morlacus** [comment]
I agree with you that's why I tried a "less mathematical" explanation. Now what if I give this explanation in the french version - Or is it necessary to have a "strict" translation - I can understand that and I will do it if necessary. 
Could be good to have more explanation or give examples of what is possible to do, vtail, heli settings sticky throttle hold (but for detailed explanation I think a dedicated manual would be necessary....)

**Comment by TheRealMoeder** [comment]
Of course you could do custom translations with further explanation, but I think it makes more sense to work on the English version first. After thinking about your proposal I haven't found a better way of putting it, so I suggest you make the changes in a fork of the manual and then do a pull request. 

Concerning additional explanations, I personally  think we should add a new section to the manual like "tips and tricks" which could explain things like sticky throttle hold and the like.
